### PR TITLE
disable a feature toggle that is on by default

### PR DESCRIPTION
**Pull request by mjseaman**
We document how to enable a feature toggle, but not how to disable one. This adds the config setting to disable a toggle that is on by default.

Would appreciate a check on whether the syntax is correct, and also confirm that this is config functionality we want users to use.

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -2220,6 +2220,10 @@ For more information about Grafana Enterprise, refer to [Grafana Enterprise]({{<
 
 Keys of alpha features to enable, separated by space.
```
Comment (Elfo404) [inline docs/sources/setup-grafana/configure-grafana/_index.md:2221]: given alpha features are not the only one that can be enabled:
```suggestion
Keys of features to enable, separated by space.
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -2220,6 +2220,10 @@ For more information about Grafana Enterprise, refer to [Grafana Enterprise]({{<
...
 
+### FEATURE_TOGGLE_NAME = false 
+  
+Some feature toggles for stable features are on by default. Use this setting to explicitly disable an on-by-default feature toggle with the name FEATURE_TOGGLE_NAME, for example `exploreMixedDatasource = false`.
```
Comment (Elfo404) [inline docs/sources/setup-grafana/configure-grafana/_index.md:2225]: we may also make this a but more explicit, like this maybe (but its pretty inconsistent compared to the rest of the page)

image

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 3]
```diff
@@ -2218,7 +2218,11 @@ For more information about Grafana Enterprise, refer to [Grafana Enterprise]({{<
 
 ### enable
 
-Keys of alpha features to enable, separated by space.
+Keys of features to enable, separated by space.
+
+### FEATURE_TOGGLE_NAME = false 
+  
+Some feature toggles for stable features are on by default. Use this setting to explicitly disable an on-by-default feature toggle with the name FEATURE_TOGGLE_NAME, for example `exploreMixedDatasource = false`.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:2225]:
```suggestion
Some feature toggles for stable features are on by default. Use this setting to disable an on-by-default feature toggle with the name FEATURE_TOGGLE_NAME, for example, `exploreMixedDatasource = false`.
```

**Review by chri2547 — APPROVED** [review]
A small suggestion. Thanks for the contribution!

File: docs/sources/setup-grafana/configure-grafana/_index.md (hunk 2 above)
Comment (ryantxu) [reply docs/sources/setup-grafana/configure-grafana/_index.md:2225]: there is similar language in:
https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L1492

that gets included in the depolyments

**Pull request merged**